src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/AddAzureRmDataLakeAnalyticsFirewallRule.cs

```diff
@@ -53,7 +53,7 @@ public class AddAzureRmDataLakeAnalyticsFirewallRule : DataLakeAnalyticsCmdletBa
         public override void ExecuteCmdlet()
         {
             ConfirmAction(
-                string.Format(Resources.AddDataLakeFirewallRule, Name),
+                string.Format(Properties.Resources.AddDataLakeFirewallRule, Name),
```

Comment:
Can you revert these changes? from Resources to Properties.Resources... Resource.resx files are not supposed to be inside properties folder...This is also causing a lot of noise in the PR (extra files)